src/ResourceManager/Network/Commands.Network/Common/NetworkCloudException.cs

```diff
@@ -51,39 +51,53 @@ protected static string GetErrorMessageWithRequestIdInfo(Rest.Azure.CloudExcepti
                 return sb.ToString();
             }
 
+            sb.AppendLine().AppendFormat("StatusCode: {0}", cloudException.Response.StatusCode.GetHashCode());
+            if(cloudException.Response.ReasonPhrase != null)
+            {
+                sb.AppendLine().AppendFormat("ReasonPhrase: {0}", cloudException.Response.ReasonPhrase);
+            }
+
             if (cloudException.Response.StatusCode.Equals(HttpStatusCode.OK)
                 && cloudException.Response.Content != null)
             {
-                
                 var errorReturned = JsonConvert.DeserializeObject<PSNetworkLongRunningOperation>(
                     cloudException.Response.Content);
-                
-                
+
                 sb.AppendLine().AppendFormat("Status: {0}", errorReturned.Status);
                 if (errorReturned.Error == null)
                 {
                     return sb.ToString();
                 }
-                
+
+                if (errorReturned.Error.Code != null)
+                {
+                    sb.AppendLine().AppendFormat("ErrorCode: {0}", errorReturned.Error.Code);
+                }
+
+                if (errorReturned.Error.Code != null)
```

Comment:
This check should be for `errorReturned.Error.Target`

Also, what information does the `Error.Target` convey?

Reply:
This seems to be similar to [this](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SdkCommon/ClientRuntime.Azure/ClientRuntime.Azure/CommonModels/CloudError.cs#L37) but supposed to be deserialized from JSON response, though there is not much description in SDK.
I also didn't manage to trigger error that adds value there to get more details. In general, we would never show it if it is not present.